app/scripts/components/data-catalog/filters-control.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useCallback } from 'react';
+import styled from 'styled-components';
+import { Taxonomy } from 'veda';
+import SearchField from '$components/common/search-field';
+import CheckableFilters, { OptionItem } from '$components/common/form/checkable-filter';
+import { Actions, useBrowserControls } from '$components/common/browse-controls/use-browse-controls';
+
+const ControlsWrapper = styled.div<{ width?: string; }>`
+  min-width: 20rem;
+  width: ${props => props.width ?? '3rem'};
```

Comment:
Am I reading right that width is smaller than min-width? then dos this style do anything?